src/opm/input/eclipse/Schedule/Well/Well.cpp

```diff
@@ -511,6 +514,15 @@ bool Well::updateEconLimits(std::shared_ptr<WellEconProductionLimits> econ_limit
     return false;
 }
 
+bool Well::updateWVFPEXP(std::shared_ptr<WVFPEXP> wvfpexp_arg) {
+    if (*this->wvfpexp != *wvfpexp_arg) {
+        this->wvfpexp = wvfpexp_arg;
```

Comment:
Another optimisation opportunity here.  Please write this as
```C++
this->wvfpexp = std::move(wvfpexp_arg);
```
instead.  That way we avoid needlessly incrementing the reference count associated to the `shared_ptr` object.